pkg/client/stomp_connection.go

```diff
@@ -19,39 +19,41 @@ import (
 	"io"
 	"net"
 
+		"github.com/golang/glog"
+
 	"github.com/go-stomp/stomp"
 )
 
 // StompConnection is an implementation of Connection interface
 type StompConnection struct {
 	// Global options:
-	brokerHost      string
-	brokerPort      int
-	destinationName string
-	userName        string
-	userPassword    string
-	useTLS          bool
-	insecureTLS     bool
+	BrokerHost      string
```

Comment:
This renaming of the fields looks like the subject for a different pull request, or at least for a different patch. Also, if we are going to use this style of constructor, then there should be a separate type, `STOMPConnectionConfiguration` or similar, so that the fields are public in that type, but private in this type. Otherwise the user of the library may think that she can change the values of this fields, when actually it will have no effect after creating the connection.

Reply:
Moved to #9